projects/distributed-tracing/src/shared/components/span-detail/call/body/span-detail-call-body.component.ts

```diff
@@ -30,7 +30,7 @@ import { ToggleViewMode } from '@hypertrace/components';
             </ht-message-display>
           </section>
           <section *ngSwitchCase="this.rawLabel">
-            <pre class="pre-body">{{ this.body | json }}</pre>
+            <pre class="pre-body">{{ this.rawBody }}</pre>
```

Comment:
Are we not pretty printing this anymore? Did this ever work - I thought the json pipe takes objects.

Reply:
Yep. Don't think it worked for raw format.